wsynphot/get_filter_data.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+import pandas as pd
+import requests
+import io
+from astropy import units as u
+
+FLOAT_MAX = np.finfo(np.float64).max
+SVO_MAIN_URL = 'http://svo2.cab.inta-csic.es/theory/fps/fps.php'
+
+def get_filter_index(wavelength_eff_min=0, wavelength_eff_max=FLOAT_MAX):
+    """Get master list (index) of all filters at SVO
+    Optional parameters can be given to get filters data for specified
+    Wavelength Eff. range from SVO
+
+    Parameters
+    ----------
+    wavelength_eff_min : float, optional
+        Minimum value of Wavelength Eff. (default is 0)
+    wavelength_eff_max : float, optional
+        Maximum value of Wavelength Eff. (default is a very large no. 
+        FLOAT_MAX - maximum value of np.float64)
+
+    Returns
+    -------
+    BytesIO object
+        VOTable fetched from SVO (in response to query) as a buffer
+    """
+    wavelength_eff_min = u.Quantity(wavelength_eff_min, u.angstrom)
+    wavelength_eff_max = u.Quantity(wavelength_eff_max, u.angstrom)
+    query = {'WavelengthEff_min': wavelength_eff_min.value,
+             'WavelengthEff_max': wavelength_eff_max.value}    
+    response = requests.get(SVO_MAIN_URL, params=query)
+    response.raise_for_status()
+    return io.BytesIO(response.content)
```

Comment:
a parsed votable should be returned - sorry was not clear on that.

Reply:
Done
Let me know if still any changes required in what is being returned?